toolkit/tools/pkg/imagecustomizerlib/baseconfigs_test.go

```diff
@@ -68,20 +70,43 @@ func TestBaseConfigsInputAndOutput_FullRun(t *testing.T) {
 		t.Skip("systemd-boot not available on AZL3 ARM64 yet")
 	}
 
-	testTmpDir := filepath.Join(tmpDir, "TestBaseConfigsInputAndOutput_FullRun")
+	testTmpDir := filepath.Join(tmpDir, "TestBaseConfigsFullRun")
 	defer os.RemoveAll(testTmpDir)
 
 	buildDir := filepath.Join(testTmpDir, "build")
-	outImageFile := filepath.Join(testTmpDir, "image.vhdx")
+	outImageFilePath := filepath.Join(testTmpDir, "image.vhdx")
 
 	currentConfigFile := filepath.Join(testDir, "hierarchical-config.yaml")
 
 	err = CustomizeImageWithConfigFile(t.Context(), buildDir, currentConfigFile, baseImage, nil,
-		outImageFile, "vhdx", true, "")
+		outImageFilePath, "vhdx", true, "")
 	if !assert.NoError(t, err) {
 		return
 	}
 
-	assert.FileExists(t, outImageFile)
+	assert.FileExists(t, outImageFilePath)
 
+	imageConnection, err := testutils.ConnectToImage(buildDir, outImageFilePath, true, coreEfiMountPoints)
```

Comment:
It looks like the test is failing:

INFO[0146][main_test] Success!                                     
    baseconfigs_test.go:90: 
        	Error Trace:	/mnt/vss/_work/azure-linux-image-tools/azure-linux-image-tools/repo/toolkit/tools/pkg/imagecustomizerlib/baseconfigs_test.go:90
        	Error:      	Received unexpected error:
        	            	failed to create mountpoints for chroot:
        	            	failed to mount (/dev/loop0p2) to (/mnt/vss/_work/azure-linux-image-tools/azure-linux-image-tools/repo/toolkit/tools/pkg/imagecustomizerlib/_tmp/TestBaseConfigsFullRun/build/testimageroot):
        	            	no such file or directory
        	Test:       	TestBaseConfigsFullRun